static/js/components/TelescopeMap.jsx

```diff
@@ -0,0 +1,210 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { useDispatch } from "react-redux";
+import {
+  ComposableMap,
+  Geographies,
+  Geography,
+  Marker,
+  useZoomPan,
+} from "react-simple-maps";
+
+const geoUrl =
+  "https://raw.githubusercontent.com/zcreativelabs/react-simple-maps/master/topojson-maps/world-110m.json";
```

Comment:
If one doesn't have access to the internet and is developing, what does the app do?

Reply:
i didn't think about this... should we directly add the svg to the data folder ? especially because its a very tiny file.

Reply:
Could you give it a try? Not quite sure about permissioning, but seems like our goal is to limit the things we must be connected to the internet for.

Reply:
Ok, so it seems to work. I even tried accessing it from another device (my phone) and as a user with the most basic skyportal rights and it seemed to work. One issue i noticed: on smaller screen and portrait mode (like a phone), or even on the computer if i resize the window by a lot, the map doesnt show up anymore.